app/assets/javascripts/widgets/brochure_masthead.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  $(".offcanvas-menu-trigger a").on("click", function(){
+    $("#js-cu-off-canvas-nav-trigger").trigger("click");
+    return false;
```

Comment:
This is for the menu button on the brochure masthead. It triggers the off canvas-nav to open. I can move this Event handler into the omninav js, but since it is only for brochure pages I thought it made more sense here. What do you think?